### PR TITLE
omit namespace to avoid ambiguity

### PR DESCRIPTION
**Pull request by donggangcj**
**What type of PR is this?**

> /area kubectl
> /kind cleanup

**What this PR does / why we need it**:This PR resolve eliminate the ambiguity `kubectl get` returnd.

**Which issue(s) this PR fixes**:https://github.com/kubernetes/kubectl/issues/757

**Special notes for your reviewer**:
/assign @deads2k @juanvallejo

**Does this PR introduce a user-facing change?**:Yes

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by zhouya0** [comment]
Thanks a lot, this might be the sulotion. But i think if we want to reserve namespace we can use a bool variable like `printWithNamespace`  in `Run` function.
Like in many resources, once we found a resource `printWithNamespace` is false, we are not going to print namespace.

**Comment by donggangcj** [comment]
@zhouya0 Thanks. The `get` command can get namespaced and nonamespaced resource at the same time。if Command `kubectl get nodes,pods --printWithNamespace=true`  return `No resouce  found in default namespace`,it is still ambiguous.It is wise to ignore the namespace information in the old version. This namespace information is of no value to the user because when necessary, the user will specify the flag `--namesapce`.
My english is pool.😂

**Comment by smarterclayton** [comment]
/ok-to-test

The message shouldn't be relying on "all-namespaces".  We might want to base the message on whether all resources listed are namespaced (which wouldn't work for files from disk) and allNamespaces is set.  What does it look like to determine whether all resource info inputs are namespaced from this part of the code?

**Comment by donggangcj** [comment]
I think it is not worth determine whether all resource info inputs are namespaced from this part of the code. Users don't seem to  care about the result message whether cotain the namespace info. 🙂

**Review by soltysh — CHANGES_REQUESTED** [review]
@donggangcj I agree with what Clayton said earlier, can you look into how much work it is not to change that message, I know the user base a bit and I know we'll get complains as soon as this lands. Also, when working with a default namespace (not `default` but one that you have hardcoded in your configfile) you don want to know the name of it if you just invoke `kubectl get` w/o specifying the namespace. That's why before we make the final call for removing this I want to know the cost.

**Comment by rccrdpccl** [comment]
Hi @donggangcj - are you still interested in working on this issue?

**Comment by rccrdpccl** [comment]
I've continued @donggangcj work at https://github.com/kubernetes/kubernetes/pull/89861 that should fix the original issue

**Pull request closed without merging**
